kernel/src/test/generate_tests.sh

```diff
@@ -4,6 +4,14 @@
 # the makefile will run this file automatically while building,
 # so all tests are automatically executed.
 
+# https://www.gnu.org/software/coreutils/manual/html_node/Random-sources.html
+get_seeded_random()
+{
+  seed="$1"
+  openssl enc -aes-256-ctr -pass pass:"$seed" -nosalt \
```

Comment:
Now we need openssl as dep. And we don't need crypto security anyway. But to prevent --random-source being too short maybe use sha256sum as it's in coreutils which is a but more likely to be on a system.

Reply:
I think that using /dev/urandom or /dev/random here would be the best option, the security of the randomness doesn't really matter and we only run the command once.

Reply:
Yea, but if you use a seed, and the seed is too short, it wouldn't work. openssl pads it to a fixed length, and so wout sha256sum. But indeed, when we're not seeding it, /dev/urandom is fine